atom:
  # Gamma parameter for RBF covariance kernel. Higher values reduce covariance at
  # a given distance.
  gamma: 2.0

  # Atomic radius.
  radius: 1.0

  # Optionally update signed distance and occupancy separately.
  update_occupancy: true
  update_signed_distance: false

  # Maximum distance to raycast into a surface (think of this as the surface
  # thickness).
  max_occupied_backoff: 2.5

  # Maximum distance to raycast into free space along the surface normal.
  max_normal_backoff: 10.0

  # Noise variance for GP signed distance function regression.
  noise: 1.0

  # Number of neighbors to examine for GP regression.
  num_neighbors: 30

  # Probability update for hits/misses.
  probability_hit: 0.7
  probability_miss: 0.3

  # Visualization topic names.
  full_occupancy_topic: occupancy
  full_sdf_topic: sdf

  # Fixed frame id.
  fixed_frame_id: world
#  fixed_frame_id: velodyne

  # Thresholds for visualizing occupied atoms and those near surfaces.
  occupied_threshold: 0.5
  sdf_threshold: 1.0

  # Boolean to visualize only those atoms that are either occupied
  # or near surfaces.
  only_show_occupied: true

  # Radius for nearest neighbor search for surface normal estimation. This should
  # be on the order of the voxel grid resolution if input point clouds are filtered.
  surface_normal_radius: 3.0